### ISSUE TITLE
Grease pencil HiDPI settings

### ISSUE THREAD

**Issue by NPDan87** [Pullusb/greasepencil_tools#4]
This is not really an issue, its more as a feature request.
Working with 2 monitors one Full HD one 4K i have a lot of moments where the cursor isn't visible due to it being to small not adapting to offer some feedback on how much the brush size. In the Full HD one i can find it eventualy, but on the 4k there are so many moments when i cannot find the cursor and i have to position it outside of the canvas to regain usability.
While on a display tablet this wont happed, on a graphic tablet its a bit anoying. 
My purposal is that changing the cursor to match the brush size at least, should be only available via this addon and not inside the whole blenders UI settings, if posible.

Thanks, for this awsome tool BTW!

**Comment by Pullusb** [comment]
Hi, when a you talk about the cursor, I guess you mean the drawing cursor ?
I don't have any controls on native blender UI concerning brush icon/crosshair with python.
If I see a good workaround to add this I'll try something but not sure really.

And there is supposedly already an options for this. Have you tried enabling the `Display cursor` ? (this make a dot cursor a bit more visible)

![image](https://user-images.githubusercontent.com/7763187/121161921-bd5da500-c84d-11eb-86d7-bd232ee7ba18.png)

Thank you for the feedback

**Comment by NPDan87** [comment]
Yes i have that enabled (its on by default) the weird thing is that the cursor icon is the one that the OS uses on the rest of blender. As soon as you enter draw mode it changes to a perfect shape to draw with on a display tablet, not so mutch on a graphical tablet and its size is pixel based, so on a higher dpi screen you can bearly see it once you loose track of it.

Thanks!
